### DISCUSSION TITLE
Share Link to All Logged in Members

### DISCUSSION THREAD

**Discussion by Abdillah** [outline/outline#3587] [Ideas]
Hello, 

There is a feature gap that arises when we want to share a doc with logged in colleagues with no permission to the collection. Activates Internet-wide sharing is security risk, while adding the person to the collection will enable the person to access other document. This feature is highlighted in Discussion #2010 but seems like not listed in issue for this use case. 

**A suggestion:** currently Outline provides either collection's members restricted link or Internet-visible link. I think the current feature of sharing the document to collection's members is–I would say humorously–controversial. It should be for all logged in members, since the original link should works and the created link means for read-only access. Or a checkbox to enable this feature is also good enough (without replacing the functionality).

That's what in my mind for Outline to be more amazing.

Regards,

@Abdillah

**Comment by Abdillah** [comment]
My particular use case is when we have internal document in a private collection that wanted to be shared with colleagues.
This document might be an announcement to the employee or something. Sharing the doc to the public is unreasonable in this case since it might leak company decision or private data.

It might also be an announcement to certain group in the company. But everyone has signed an NDA, so a leak inside the company sometime is tolerable or more controllable.

**Reply by tommoor** [reply]
I believe this is covered by #2359  – let me know if that's right

**Reply by Abdillah** [reply]
That's exactly what we need :+1: 
Thanks.
